### PR TITLE
Toast - fix typings

### LINKED ISSUE #1249: "Toast" doc may need to be updated
### Describe the bug
The props described in the doc does not match the component's props.
image
image

### Component?
Toast

### Is there a general issue with the docs site?
No

### Is there an issue with the Expo snack app?
As the snack app not able to run ...... I guess yes ....

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Toast - fix typings

## Changelog
Toast - fix typings

Resolves #1249

### CODE REVIEW

File: typings/components/Toast.d.ts [hunk 1]
```diff
@@ -19,18 +12,16 @@ export interface ToastProps {
   backgroundColor?: ColorValue;
   color?: ColorValue;
   message?: string;
-  messageStyle?: StyleProp<TextStyle>;
   icon?: ImageRequireSource;
-  actions?: ReadonlyArray<ButtonProps>;
```
Comment (lidord-wix) [inline typings/components/Toast.d.ts:24]: `action` prop is missing

File: typings/components/Toast.d.ts [hunk 2]
```diff
@@ -19,18 +12,16 @@ export interface ToastProps {
   backgroundColor?: ColorValue;
```
Comment (lidord-wix) [inline typings/components/Toast.d.ts:12]: `height` prop is unused :)

**Review by lidord-wix — CHANGES_REQUESTED** [review]

File: typings/components/Toast.d.ts (hunk 1 above)
Comment (M-i-k-e-l) [reply typings/components/Toast.d.ts:24]: Fixed

File: typings/components/Toast.d.ts (hunk 2 above)
Comment (M-i-k-e-l) [reply typings/components/Toast.d.ts:12]: Fixed

File: typings/components/Toast.d.ts [hunk 3]
```diff
@@ -1,36 +1,28 @@
...
   position?: ToastPosition;
-  height?: number;
   backgroundColor?: ColorValue;
   color?: ColorValue;
   message?: string;
-  messageStyle?: StyleProp<TextStyle>;
   icon?: ImageRequireSource;
-  actions?: ReadonlyArray<ButtonProps>;
-  onDismiss?: (event: GestureResponderEvent) => void;
+  actions?: ButtonProps;
```
Comment (lidord-wix) [inline typings/components/Toast.d.ts:16]: Should be `action`, no?

Comment (M-i-k-e-l) [reply typings/components/Toast.d.ts]: Oops, fixed

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**